CodeStyle.Analyzers/CodeStyle.Analyzers.csproj

```
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+        <IsPackable>true</IsPackable>
+        <RootNamespace>Messerli.CodeStyle.Analyzers</RootNamespace>
+        <LangVersion>8.0</LangVersion>
```

Use the newest language version
```suggestion
        <LangVersion>9.0</LangVersion>
```